sa/model.go

```diff
@@ -212,11 +212,11 @@ type issuedNameModel struct {
 
 // regModel is the description of a core.Registration in the database before
 type regModel struct {
-	ID        int64    `db:"id"`
-	Key       []byte   `db:"jwk"`
-	KeySHA256 string   `db:"jwk_sha256"`
-	Contact   []string `db:"contact"`
-	Agreement string   `db:"agreement"`
+	ID        int64  `db:"id"`
+	Key       []byte `db:"jwk"`
+	KeySHA256 string `db:"jwk_sha256"`
+	Contact   []byte `db:"contact"`
```

Comment:
Why `[]byte` rather than `string` here?

Reply:
Because prior to resolving your previous comment, I was relying on the ability for it to be truly `nil` to take advantage of `json.Marshal`'s semantics. Now that we're not doing that, `string` is better, you're right.

Reply:
Oh, and because `[]byte` is the default type for json.(Un)Marhsal. Converting back and forth between strings introduces more conversions at that API boundary. Take a look.